### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by fdiaz**
Updates links incorrectly pointing to the wrong repo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ This document explains some of the guidelines around contributing to this projec
 
 ## Alignment first
 
-We ask you that if you have an idea for a new feature you [open an issue](https://git.musta.ch/francisco-diaz/SwiftInspector/issues/new) to discuss or reach out to [francisco-diaz](https://git.musta.ch/francisco-diaz) to align first.
+We ask you that if you have an idea for a new feature you [open an issue](https://github.com/fdiaz/SwiftInspector/issues/new).
```
Comment (dfed) [inline CONTRIBUTING.md:9]: could this be
```suggestion
We ask you that if you have an idea for a new feature you [open an issue](issues/new).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ This document explains some of the guidelines around contributing to this projec
...
 
 ## Submitting your changes
 
-Whenever your feature is ready for review, please [open a PR](https://git.musta.ch/francisco-diaz/SwiftInspector/pull/new/master) with a clear list of what you've done.
+Whenever your feature is ready for review, please [open a PR](https://github.com/fdiaz/SwiftInspector/pull/new/master) with a clear list of what you've done.
```
Comment (dfed) [inline CONTRIBUTING.md:13]: Could this be
```suggestion
Whenever your feature is ready for review, please [open a PR](pull/new/master) with a clear list of what you've done.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (fdiaz) [reply CONTRIBUTING.md:9]: Maybe? Let's try it out.

File: CONTRIBUTING.md (hunk 2 above)
Comment (fdiaz) [reply CONTRIBUTING.md]: Updated https://github.com/fdiaz/SwiftInspector/pull/34/files#diff-6a3371457528722a734f3c51d9238c13

File: CONTRIBUTING.md (hunk 1 above)
Comment (fdiaz) [reply CONTRIBUTING.md]: Works like this https://github.com/fdiaz/SwiftInspector/pull/34/files#diff-6a3371457528722a734f3c51d9238c13

File: CONTRIBUTING.md (hunk 2 above)
Comment (dfed) [reply CONTRIBUTING.md]: Sweet!

**Pull request merged**